server/lib/search.js

```diff
@@ -60,12 +60,17 @@ export const searchCollectivesByEmail = async (email, user, offset = 0, limit =
 };
 
 /**
- * Turn a search string into a TS vector using 'OR' operator.
+ * Sanitize and then turn a search string into a TS vector using 'OR' operator.
+ *
+ * Examples: "open potatoes" => "open|potatoes", "crème brulée => "creme|brulee"
  *
- * Ex: "open potatoes" => "open|potatoes"
  */
 export const searchTermToTsVector = term => {
-  return term.replace(/\s+/g, '|');
+  const sanitizedTerm = term
+    .normalize('NFD')
+    .replace(/[\u0300-\u036f]/g, '')
+    .replace(/[^a-zA-Z0-9-\/_ ]/g, '');
```

Comment:
We should add a comment to explain this one. We both know what this is, but the next person who's gonna look at this code will have a hard time understanding this. Plus variable name is not ideal, we're not really sanitizing here, just removing the accents. I can suggest `termWithoutAccents`

Reply:
The proper term I think is [Diacritic](https://en.wikipedia.org/wiki/Diacritic). So I renamed this to, `termWithoutDiacritics`.